src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -11,5 +11,16 @@ public class CliSyntax {
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_GENDER = new Prefix("g/");
+    public static final Prefix PREFIX_BIRTHDAY = new Prefix("b/");
+    public static final Prefix PREFIX_IC_NUMBER = new Prefix("i/");
+    public static final Prefix PREFIX_DEPARTMENT = new Prefix("d/");
+
+    // Prefixes for Record
+    public static final Prefix PREFIX_OBSERVATION = new Prefix("o/");
```

Comment:
Would it better to name it `PREFIX_INITIAL_OBSERVATIONS` instead? This is so that it will be in line with the initialObservations field in `Record.java` class.